opendevin/controller/agent_controller.py

```diff
@@ -292,32 +295,36 @@ async def step(self, i: int) -> bool:
     def get_state(self):
         return self.state
 
-    def _is_stuck(self):
+    def _is_stuck(self, is_delegate=False):
```

Comment:
`delegate` is just another `AgentController`. So we should be able to just say:
```
if self.delagate._is_stuck:
  return True
# rest of state checking logic
```

Reply:
Thanks, it's modified.